Remove classgraph dependency from sbt plugin scope

scala-steward opened the pull request:
Updates [io.github.classgraph:classgraph](https://github.com/classgraph/classgraph) from 4.8.47 to 4.8.53.


I'll automatically update this PR to resolve conflicts as long as you don't change it yourself.

If you'd like to skip this version, you can just close this PR. If you have any feedback, just mention me in the comments below.

Have a fantastic day writing Scala!

<details>
<summary>Ignore future updates</summary>

Add this to your `.scala-steward.conf` file to ignore future updates of this dependency:
```
updates.ignore = [ { groupId = "io.github.classgraph", artifactId = "classgraph" } ]
```
</details>

labels: semver-patch

scala-steward committed:
Update classgraph to 4.8.53

raboof reviewed (commented):
I don't think we need this anymore because we're using sbt-paradox-unidoc (via sbt-paradox-akka) now - let me check.

raboof committed:
Remove classgraph dependency

As we don't use it directly and it comes in transitively from
sbt-paradox-unidoc where it is used.

raboof reviewed (commented):
Jup looks OK, removed it.

raboof commented:
OK TO TEST

patriknw reviewed (approved):
LGTM, less is more
